images/build/Dockerfiles/full.Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN ./build.sh golang     /opt/startupcmdgen/golang
 FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
 ARG DEBIAN_FLAVOR
 ARG SDK_STORAGE_BASE_URL_VALUE="https://oryx-cdn.microsoft.io"
-ARG AI_KEY
+ARG AI_CONNECTION_STRING
+ENV ORYX_AI_CONNECTION_STRING=${AI_CONNECTION_STRING}
```

Comment:
There's a lingering `ORYX_AI_INSTRUMENTATION_KEY="${AI_KEY}"` environment variable being set at the bottom of this file